pages/docs/reference/mpp-share-on-all-platforms.md

```diff
@@ -0,0 +1,56 @@
+---
+type: doc
+layout: reference
+title: "Discover your project"
+---
+# Share code on all platforms
+
+If you have business logic that is common for all platforms, you don’t need to write the same code for each platform – 
+just share it in the common source set.
+
+![Code shared for all platforms]({{ url_for('asset', path='images/reference/mpp/flat-structure.png') }})
+
+All platform-specific source sets depend on the common source set by default. You don’t need to specify any `dependsOn` 
+relations manually for default source sets, such as `jvmMain`, `macosX64Main`, and others. 
+
+If you add a custom source set and want to use dependencies of `commonMain` and reference its declarations, add the 
```

Comment:
I'm not sure this recommendation is in the right place here, as just creating the source set without adding it to the compilations won't do anything.

Reply:
Removed from the picture and text.